tests/e2e/multi_svc_test.go

```diff
@@ -0,0 +1,618 @@
+	var (
+		namespaces          []string
+		numberOfSvc         int
+		clients             []clientset.Interface
+		storagePolicyNames  [2]string
+		scParametersList    [2]map[string]string
+		scParameters        map[string]string
+		storageClassNames   [2]string
+		csiNamespace        string
+		dataCenter          string
+		kubeconfig1         string
+		computeClusterPaths []string
+		vcAddress           string
+		supervisorIds       []string
+		wcpServiceAccUser   []string
+	)
+	ginkgo.BeforeEach(func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		numberOfSvc = 2
+		clients, namespaces = getMultiSvcClientAndNamespace()
+		envStoragePolicyNameForSharedDatastoresList := []string{envStoragePolicyNameForSharedDatastores, envStoragePolicyNameForSharedDatastores1}
+		for i := 0; i <= numberOfSvc-1; i++ {
+			storagePolicyNames[i] = GetAndExpectStringEnvVar(envStoragePolicyNameForSharedDatastoresList[i])
+			setResourceQuota(clients[i], namespaces[i], rqLimit)
+			scParameters = make(map[string]string)
+			scParametersList[i] = scParameters
+		}
+
+		bootstrap()
+
+		for i := 0; i <= numberOfSvc-1; i++ {
+			nodeList, err := fnodes.GetReadySchedulableNodes(clients[i])
+			framework.ExpectNoError(err, "Unable to find ready and schedulable Node")
+			if !(len(nodeList.Items) > 0) {
+				framework.Failf("Unable to find ready and schedulable Node")
+			}
+		}
+
+		csiNamespace = GetAndExpectStringEnvVar(envCSINamespace)
+		computeClusterPaths = getSvcComputeClusterPath()
+		vcAddress = e2eVSphere.Config.Global.VCenterHostname + ":" + sshdPort
+		dataCenter = GetAndExpectStringEnvVar(datacenter)
+		kubeconfig1 = GetAndExpectStringEnvVar("KUBECONFIG1")
+		framework.Logf("datacenter name is : %v", dataCenter)
+
+		ginkgo.By("Getting User and Supervisor-Id for both the supervisors")
+		for i := 0; i <= numberOfSvc-1; i++ {
+			currentSecret, err := clients[i].CoreV1().Secrets(csiNamespace).Get(ctx, configSecret, metav1.GetOptions{})
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			originalConf := string(currentSecret.Data[vsphereCloudProviderConfiguration])
+
+			vsphereCfg, err := readConfigFromSecretString(originalConf)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			supervisorIds = append(supervisorIds, vsphereCfg.Global.SupervisorID)
+			wcpServiceAccUser = append(wcpServiceAccUser, strings.Split(string(vsphereCfg.Global.User), "@")[0])
+		}
+
+	})
+
+	ginkgo.AfterEach(func() {
+		framework.Logf("log-after-each-1")
+		/*
+			clients, namespaces = getMultiSvcClientAndNamespace()
+			for i := 0; i <= numberOfSvc-1; i++ {
+				framework.Logf("log-after-each-2-1")
+				fss.DeleteAllStatefulSets(clients[i], namespaces[i])
+				framework.Logf("log-after-each-2-2")
+				setResourceQuota(clients[i], namespaces[i], defaultrqLimit)
+				framework.Logf("log-after-each-2-3")
+				dumpSvcNsEventsOnTestFailure(clients[i], namespaces[i])
+				framework.Logf("log-after-each-2-4")
+			}
+		*/
```

Comment:
remove unnecessary code, or add a comment when you need to check this in...

Reply:
removed